### PR TITLE
More menu ui

### LINKED ISSUE #538: More UI: 2.0
https://gallery.io/projects/MCHbtQVoQ2HCZXGcfotOfnw9/files/MCEoODg2_SOQVaJniPl2OLRm

### LINKED ISSUE #594: Wallet Page (Sign message)
Link to views handling message signing. Message signing should be confirmed by the user after entering the wallet PIN/Password

Currently blocked by #559 

References: 
1.  https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDSccR5VbIX_PdlHrPm6QgJC5c
2. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScQxQ_251v6mfFksadZ7Ca7s
3. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScQjm_Hl3dKB6-xA29OAq0Xw
4. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScY6h7f0suIUOuHRMfNVak7s

After successful singing:
1. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScaY7C4szgl_5SGqsbE0yXxM
2. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScclXfga9t7Jn9y-IUKnYoL0

### LINKED ISSUE #595: Wallets page (Verify message, address and signature)
message and signature verification for a wallet address
currently blocked by https://github.com/raedahgroup/dcrios/pull/559

Link `verify message` buttons to present these screens
1. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScaXS5JOYPz4qFF8LcbnkmOg
2. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScYYLAm1odrwfpGVfb0AJfo0
3. https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScdpDl-iZXxio67eDUI0IMuU
4. valid signature: https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScWI1ReugSIndj9w0QjPFzUY
5. invalid signature: https://gallery.io/projects/MCHbtQVoQ2HCZZTQVworZ4Kv/files/MCEJu8Y2hyDScd4G1JiksEXOsiMDH82S34A

### PR DESCRIPTION
**Pull request by macsleven**
This PR separates all the screens involve into its own storyboard which makes editing and reviewing easier.
Each menu screen can be worked on with different PR since they are now independent.
closes #538 
closes #595
closes #594 

More Menu

Screenshot 2020-01-06 at 19 21 28

Settings Menu:

Screenshot 2020-01-06 at 19 21 42

Security Tools Menu

Screenshot 2020-01-06 at 19 21 54

Validate Address

Screenshot 2020-01-06 at 19 22 13

Sign Message

Screenshot 2020-01-06 at 19 24 41

Sign Message Page Info

Screenshot 2020-01-06 at 19 24 51

Screenshot 2020-01-06 at 19 25 35

Validate address
Screenshot 2020-01-06 at 19 26 10
Screenshot 2020-01-06 at 19 29 52
Screenshot 2020-01-06 at 19 30 05

Verify message
Screenshot 2020-01-06 at 19 53 05

Screenshot 2020-01-06 at 19 53 59

Screenshot 2020-01-06 at 19 53 25

Screenshot 2020-01-06 at 19 31 28

Screenshot 2020-01-06 at 19 31 39

Screenshot 2020-01-06 at 19 31 50

Screenshot 2020-01-06 at 19 32 36

Screenshot 2020-01-06 at 19 32 36

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,7 +30,7 @@ project.xcworkspace
 
 # Android/IntelliJ
 #
-build/
+/build/
```
Comment (itswisdomagain) [inline .gitignore]: No need to repeat the `/build/` ignore directive. This can be deleted.
```suggestion
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1177,7 +1317,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIconTestnet;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = LYZ95J7356;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -1188,7 +1328,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.decred.dcrios.testnet3;
+				PRODUCT_BUNDLE_IDENTIFIER = com.macsleven.dcrios.testnet3;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = com.decred.dcrios.testnet3;
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1260,7 +1400,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIconTestnet;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = LYZ95J7356;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -1271,7 +1411,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.decred.dcrios.testnet3;
+				PRODUCT_BUNDLE_IDENTIFIER = com.macsleven.dcrios.testnet3;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = com.decred.dcrios.testnet3;
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -1404,7 +1544,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = LYZ95J7356;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -1415,7 +1555,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.decred.dcrios.mainnet;
+				PRODUCT_BUNDLE_IDENTIFIER = com.macsleven.dcrios.mainnet;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				PRODUCT_BUNDLE_IDENTIFIER = com.decred.dcrios.mainnet;
```

File: Decred Wallet.xcodeproj/project.pbxproj [hunk 8]
```diff
@@ -1430,7 +1570,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = LYZ95J7356;
```
Comment (itswisdomagain) [inline Decred Wallet.xcodeproj/project.pbxproj]: Undo this change:
```suggestion
				DEVELOPMENT_TEAM = "";
```

File: Decred Wallet/Features/More/More Features/Help/HelpTableViewController.swift [hunk 9]
```diff
@@ -0,0 +1,59 @@
+//
+//  HelpTableViewController.swift
+//  Decred Wallet
+
```
Comment (quacklabs) [inline Decred Wallet/Features/More/More Features/Help/HelpTableViewController.swift]: remove empty space

File: Decred Wallet/Features/More/More Features/Security Tools/SecurityToolsViewController.swift [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+       }
+    
+    @objc func pageInfo(){
+        let alertController = UIAlertController(title: LocalizedStrings.securityTools, message: LocalizedStrings.securityToolsInfo, preferredStyle: UIAlertController.Style.alert)
+        alertController.addAction(UIAlertAction(title: LocalizedStrings.gotIt, style: UIAlertAction.Style.default, handler: nil))
+        self.present(alertController, animated: true, completion: nil)
+    }
+       
+       func navigateSecurityPage(to menuItem: SecurityToolsItem) {
+           //self.tabBarController?.present(menuItem.viewController, animated: true, completion: nil)
```
Comment (quacklabs) [inline Decred Wallet/Features/More/More Features/Security Tools/SecurityToolsViewController.swift]: unused and should be removed

File: Decred Wallet/Features/More/More Features/Security Tools/Sign Message/SignMessageViewController.swift [hunk 11]
```diff
@@ -0,0 +1,256 @@
+//
+//  SignMessageViewController.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 30/12/2019.
```
Comment (quacklabs) [inline Decred Wallet/Features/More/More Features/Security Tools/Sign Message/SignMessageViewController.swift]: copyright:
// Copyright (c)2019 The Decred developers
// Use of this source code is governed by an ISC
// license that can be found in the LICENSE file.

File: Decred Wallet/Features/More/More Features/Security Tools/Validate Addresses/ValidateAddressesViewController.swift [hunk 12]
```diff
@@ -0,0 +1,148 @@
+//
+//  ValidateAddressesViewController.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 25/12/2019.
```
Comment (quacklabs) [inline Decred Wallet/Features/More/More Features/Security Tools/Validate Addresses/ValidateAddressesViewController.swift]: change to correct copyright format.

File: Decred Wallet/Features/More/More Features/Settings/SettingsController.swift [hunk 13]
```diff
@@ -68,12 +67,17 @@ class SettingsController: UITableViewController  {
         super.viewWillAppear(animated)
         
         self.navigationController?.navigationBar.isHidden = false
+        self.navigationController?.navigationBar.tintColor = UIColor(hex: "#091440") //move to color file
```
Comment (quacklabs) [inline Decred Wallet/Features/More/More Features/Settings/SettingsController.swift]: should be `UIColor.appColors.darkblue`

File: Decred Wallet/Features/More/More Features/Settings/SettingsController.swift [hunk 14]
```diff
@@ -151,20 +150,26 @@ class SettingsController: UITableViewController  {
...
-            return LocalizedStrings.debug.capitalized
+            return LocalizedStrings.connection.capitalized
         default:
             return ""
         }
     }
     
+    override func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int){
+        let header = view as! UITableViewHeaderFooterView
+        header.textLabel?.textColor = UIColor(hex: "#3d5873")// move to color file
```
Comment (quacklabs) [inline Decred Wallet/Features/More/More Features/Settings/SettingsController.swift]: now called UIColor.appColors.darkBluishGray`

File: Decred Wallet/Features/More/MoreMenuItem.swift [hunk 15]
```diff
@@ -0,0 +1,61 @@
+//
+//  MoreMenuItem.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 03/12/2019.
```
Comment (quacklabs) [inline Decred Wallet/Features/More/MoreMenuItem.swift]: Copyright

File: Decred Wallet/Features/More/MoreMenuItem.swift [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+enum MoreMenuItem: String, CaseIterable {
+    case settings = "Settings"
+    case security = "Security"
```
Comment (quacklabs) [inline Decred Wallet/Features/More/MoreMenuItem.swift]: rawvalue should be `Security Tools`

File: Decred Wallet/Features/More/MoreMenuViewController.swift [hunk 17]
```diff
@@ -0,0 +1,54 @@
+//
+//  MoreViewController.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 26/12/2019.
```
Comment (quacklabs) [inline Decred Wallet/Features/More/MoreMenuViewController.swift]: use correct copyrights

File: Decred Wallet/Features/More/MoreMenuViewController.swift [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+    @IBOutlet weak var moreMenuTableView: UITableView!
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        let emptyButton = UIBarButtonItem.init()
+        let barButtonImg = UIBarButtonItem(image: UIImage(named: "ic_decred")?.withRenderingMode(.alwaysOriginal),
+        style: .done, target: self,
+        action: nil)
+        let barButtonTitle = UIBarButtonItem(title: LocalizedStrings.more, style: .plain, target: self, action: nil)
+        barButtonTitle.tintColor = UIColor(hex: "#091440") // UIColor.appColor.darkblue
```
Comment (quacklabs) [inline Decred Wallet/Features/More/MoreMenuViewController.swift]: revert to use color constants `UIColor.appColors.darkBlue`

File: Decred Wallet/Features/More/MoreMenuViewController.swift [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+        
+        self.navigationItem.leftBarButtonItems = [emptyButton, emptyButton, barButtonImg, barButtonTitle]
```
Comment (quacklabs) [inline Decred Wallet/Features/More/MoreMenuViewController.swift]: perhaps we could consider using constraints or initializing with a valid `CGPoint` rather than adding empty components as padding?

File: Decred Wallet/Features/More/More Features/Security Tools/Sign Message/SignMessageViewController.swift (hunk 11 above)
Comment (itswisdomagain) [reply Decred Wallet/Features/More/More Features/Security Tools/Sign Message/SignMessageViewController.swift]: 2020

File: Decred Wallet/Features/More/MoreMenuItem.swift (hunk 16 above)
Comment (itswisdomagain) [reply Decred Wallet/Features/More/MoreMenuItem.swift]: Should just rename the entire enum case:
```suggestion
    case securityTools = "Security Tools"
```

File: Decred Wallet/Features/More/MoreMenuViewController.swift (hunk 19 above)
Comment (itswisdomagain) [reply Decred Wallet/Features/More/MoreMenuViewController.swift]: Or use a custom view for navigation bar.
